easybuild/easyconfigs/g/GEOS/GEOS-3.11.1-GCC-12.2.0.eb

```diff
@@ -0,0 +1,20 @@
+easyblock = 'ConfigureMake'
```

Comment:
See https://github.com/easybuilders/easybuild-easyconfigs/blob/develop/easybuild/easyconfigs/g/GEOS/GEOS-3.10.3-GCC-11.3.0.eb

Reply:
@branfosj would you prefer that @jfgrimm changes this to match the earlier one?

Reply:
I've switched it to use cmake, which fixed that build failure